docs/api/doc-block-argtypes.md

```diff
@@ -0,0 +1,71 @@
+---
+title: 'ArgTypes'
+---
+
+The `ArgTypes` block can be used to show a static table of [arg types](./argtypes.md) for a given component, as a way to document its interface.
+
+<div class="aside">
+
+💡 If you’re looking for a dynamic table that shows a story’s current arg values for a story and supports users changing them, see the [`Controls`](./doc-block-controls.md) block instead.
+
+</div>
+
+![Screenshot of ArgTypes block](TK)
+
+<!-- prettier-ignore-start -->
+```md
+{/* ButtonDocs.mdx */}
+import { ArgTypes } from '@storybook/blocks';
+import * as ButtonStories from './Button.stories';
+
+<ArgTypes of={ButtonStories} />;
+```
+<!-- prettier-ignore-end -->
+
+## API
+
+### ArgTypes
+
+```js
+import { ArgTypes } from '@storybook/blocks';
+```
+
+`ArgTypes` is a React component which accepts props of type `ArgTypesProps`.
+
+<div class="aside">
+
+ℹ️ Like most blocks, the `ArgTypes` block can both be configured via props when using it directly in MDX, or with properties in `parameters.docs.argTypes`.
+
+</div>
+
+#### `ArgTypesProps`
```

Comment:
Personally I feel the pattern of documenting `ArgTypes` separately from `ArgTypesProps` is very redundant (in all cases, not just for `ArgTypes`. I don't really see how this adds any value/clarification, compared to this simplification:

```suggestion

`ArgTypes` is a React component which accepts the following props:

<div class="aside">

ℹ️ Like most blocks, the `ArgTypes` block can both be configured via props when using it directly in MDX, or with properties in `parameters.docs.argTypes`.

</div>

```

Reply:
This is largely driven by pages like `/docs/api/doc-block-colorpalette`, which contains both `ColorPalette` and `ColorItem`, each with their own props.

However, I now see why your Notion examples used "Options" as a heading (to be inclusive of both props and parameters uses), and I'll be switching to that.